src/components/MapPanel.vue

```diff
@@ -937,33 +928,228 @@ export default {
-    .leaflet-draw.leaflet-control {
-      clear: unset;
-      float: right;
+    .leaflet-touch .leaflet-draw-actions {
+      left: 230px;
     }
 
-    .leaflet-draw-toolbar.leaflet-bar.leaflet-draw-toolbar-top {
-      width: 49px;
-      height: 49px;
+    .leaflet-bar button {
+      padding: inherit !important;
     }
 
-    .leaflet-touch .leaflet-draw-actions {
-      left: 60px;
+
+          //CSS for search buttons
+
+    .leaflet-bar.leaflet-draw-toolbar>a.leaflet-draw-draw-polygon {
+      border-radius: 0;
     }
 
-    .leaflet-bar button {
-      padding: inherit !important;
+    .leaflet-bar.leaflet-control.buffer-control {
+      button {
+        border-radius: 0;
+      }
+      :focus {
+        outline: none;
+      }
+
     }
+    .leaflet-control-container>div{
+      width: 100%;
+      // .leaflet-control{
+      //   width: inherit;
+      // }
+      .pvm-container-non-mobile {
+        &:hover:after {
+          height: 100%;
+          width: 100%;
+          min-width: 315px;
+          position: absolute;
+          left: 295px;
+          content: "Type an address, owner name, property account number, or Department of Records registry map number.";
+          background: #d3d3d3;
+          opacity: 0.7;
+          display: flex;
+          // align-items: center;
+          color: #000;
+          font-style: italic;
+          padding: 4px;
+          // width: 183%;
+        }
+      }
+    }
+
+    div.buffer-control.leaflet-bar.inactive-buffer-button {
+      &:hover:after {
+        content: "Select a point on the map to show all parcels within 250-foot radius.";
+        background: #d3d3d3;
+        opacity: 0.7;
+        display: flex;
+        // align-items: center;
+        color: #000;
+        font-style: italic;
+        padding: 4px;
+        width: 183%;
+      }
+    }
+
+    div.buffer-control.leaflet-bar {
+      display: flex;
+      border: none;
 
+      button.inactive.pointer {
+        background-color: color(dark-ben-franklin);
+        span>svg {
+          color: white;
+        }
+      }
+      button.active {
+          background-color: white;
+          span>svg { color: color(dark-ben-franklin);
+          }
+          &:after {
+            background: white;
+            color: color(dark-ben-franklin);
+          }
+      }
+      button {
+        min-width: 198px;
+        display: flex;
+        span {
+          margin-left: 5px;
+          }
+        &:hover{
+          display: flex;
+        }
+        &:after {
+          font-weight: normal;
+          padding: 0 10px 0 10px;
+          content: "Select Buffer";
+          position: relative;
+          display: flex;
+          color: white;
+          // background: color(dark-ben-franklin);
+          display: flex;
+          align-items: center;
+          padding: 5px 10px 5px 10px;
+          text-transform: uppercase;
+          font-size: 16px;
+          line-height: 35px;
```

Comment:
Can you share these styles on both `:after` elements here (and in the search bar) by adding a class to them or some other way to target them?

Reply:
There's a lot of css that can be shared. Not doing that may be ok in the short term but makes it much more likely in the long term that we may have drift

Reply:
Commit added that combines a lot of the CSS